Client/src/Components/DatasetUpload/MetaSection/MetaForm.tsx

```diff
@@ -1,32 +1,50 @@
 import { Box, Grid, ThemeProvider, Typography } from '@material-ui/core'
 import { FastField, Field, FieldArray } from 'formik'
+import { ICategoryModel, ISubcategoryModel } from '../../../Remote/Endpoints/CategoryEndpoint'
 import { MuiSelectFormik, MuiTextFieldFormik } from '../../Forms/FormikFields'
 import { disabledTheme, shouldComponentUpdate } from '../../Forms/ComponentUpdate'
+import { get, values } from 'lodash'
 
 import { IMaterial } from '../../../Models/Datasets/IDatasetModel'
 import { MaterialSelectChipArray } from './MaterialSelectChipArray'
-import React from 'react'
+import React, { useState } from 'react'
 import { classStyles } from '../../../appTheme'
-import { get } from 'lodash'
 
 interface IProps {
   materials: IMaterial[],
-  categories: any[],
-  subcategories: any[],
-  editable: boolean
+  categories: ICategoryModel[],
+  editable: boolean,
+  formProps
```

Comment:
see if you can use the hooks inside this component: https://formik.org/docs/api/useFormikContext
https://formik.org/docs/api/useFormik instead of passing form props down, it should be as simple

Reply:
![image](https://user-images.githubusercontent.com/22376976/110215607-b43cf080-7e78-11eb-97ac-a096967d938c.png)

Reply:
Oh, that's way simpler. Okay, I'll change it to use useFormikContext.